### DISCUSSION TITLE
Minor issue: theme not marked as downloaded

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#377] [Q&A] [answered]
I use the Android download manager. The theme Elevate is downloaded but not marked with blue color. The downloaded OAM msps are marked blue = downloaded.

**Comment by kds112** [comment]
<img src="https://github.com/devemux86/cruiser/assets/138766725/c42c230f-7885-40a9-ba8b-9a3d1fc51918" width="50%"/>

**Reply by kds112** [reply]
<img src="https://github.com/devemux86/cruiser/assets/138766725/af4d8fdb-e109-4252-8451-90b6457cb4df" width="50%"/>

**Comment by devemux86** [comment]
@kds112 thanks for the report.

Did you download the theme with the download manager?
Does it appear in the 2nd tab with the local downloaded files?
(image 2)

Have you modified anything in the folder of the downloads?

Or did you open the theme with the "Open map theme" menu?

When using the menu "Map theme", is it in the themes list?
(image 3)

<img src="https://github.com/devemux86/cruiser/assets/3484020/f32ddd75-5db4-4c5a-8a42-5da571f8f4cb" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/cab404a3-2970-4886-9bf0-2891b5a5f309" width="33%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/c5974bb5-a712-4e17-9902-677dc95d1b20" width="33%"/>

**Comment by kds112** [comment]
> Did you download the theme with the download manager? Does it appear in the 2nd tab with the local downloaded files? (image 2)
> 
I have downloaded the theme several weeks ago. Yes, it is marked in the second Tab. 
> Have you modified anything in the folder of the Downloads? 
No. 
> 
> Or did you open the theme with the "Open map theme" menu?
> 
No. 
> When using the menu "Map theme", is it in the themes list? (image 3)

Yes, it is. 

Maybe, it is a problem that the theme is downloaded with the previous version of Cruiser?

**Reply by devemux86** [reply]
Something may be wrong in the download's properties and it is not recognized.

@kds112 can you delete it in the 2nd "local" tab and download it again?

**Reply by kds112** [reply, marked as answer]
Okay, after deleting and reinstall, it looks correct. Problem solved 👍
